src/AggregateFunctions/AggregateFunctionMinMaxAny.h

```diff
@@ -706,7 +706,7 @@ class AggregateFunctionsSingleValue final : public IAggregateFunctionDataHelper<
 public:
     AggregateFunctionsSingleValue(const DataTypePtr & type_)
         : IAggregateFunctionDataHelper<Data, AggregateFunctionsSingleValue<Data>>({type_}, {})
-        , type(this->argument_types[0])
+        , type(type_)
```

Comment:
Why it stores type and serialization if they are stored in parent argument types ? We can get type and serialization if we need  just get base argument types.

Reply:
Fixed.